snowpack/scripts/postbundle.js

```diff
@@ -0,0 +1,14 @@
+// HACK to bypass Rollup re-writing valid dynamic `import()` statement
+const fs = require('fs');
+const path = require('path');
+const rootDirectory = path.join(__dirname, '../..');
+
+function replaceNativeImport() {
```

Comment:
Don't love this but there's no way to get Rollup to ignore a particular `import`, so we're just replacing the `NATIVE_IMPORT` marker with `import` in the bundled output.

Reply:
Is there any way to move this into `generate-bundle.js`? Ex: does Rollup support an "it's over!" hook where we could run this? It would be great to not split the build in two unnecessarily.

Reply:
Ooh good question. I will look into it.